### DISCUSSION TITLE
Next release target date?

### DISCUSSION THREAD

**Discussion by buckaroogeek** [opencontainers/runc#4503] [General]
Greetings

Is there a planned release target for the next release?  I help package runc for Fedora and have been struggling with getting runc-dmz to pass integration tests using Fedora centric build flags. I am pleased to see the recent commit to remove runc-dmz.  I was wondering if a target date has been set.

thank you

**Comment by cyphar** [comment]
https://github.com/opencontainers/runc/releases/tag/v1.2.1

**Comment by buckaroogeek** [comment]
Thank you
